model/api4.py

```
@@ -87,4 +87,5 @@ def home():
     return jsonify({'message': 'Potato Disease Classification API is running'})
 
 if __name__ == '__main__':
-    app.run(debug=True, port=5000)
\ No newline at end of file
+    app.run(debug=True, port=5000)
```

Ruff (0.12.2)

Use of `debug=True` in Flask app detected

(S201)